lib/src/action.dart

```diff
@@ -41,7 +43,7 @@ import 'dart:async';
 /// when a consumer needs to check state changes immediately after invoking an
 /// action.
 ///
-class Action<T> implements Function {
```

Comment:
Yeah, we resolved it. I prefer the mixin because it leaves the super class open for future use without implying anything about a possible constructor on Disposable. More obvious for future us.